### PR TITLE
Migrate Upgrades tests

### PR DESCRIPTION
**Pull request by immrsd**
Fixes [#1011](https://github.com/OpenZeppelin/cairo-contracts/issues/1011)

### CODE REVIEW

File: src/tests/upgrades/test_upgradeable.cairo
Comment (andrew-fleming) [inline src/tests/upgrades/test_upgradeable.cairo:1]: `UpgradesV1` can be removed (line 2)

File: src/tests/upgrades/test_upgradeable.cairo [hunk 1]
```diff
@@ -59,26 +59,30 @@ fn test_new_selector_after_upgrade() {
...
 #[test]
 fn test_remove_selector_passes_in_v1() {
-    let v1 = deploy_v1();
+    let (v1, _v1_class, _v2_class) = setup_test();
+
     v1.remove_selector();
 }
 
 #[test]
+#[ignore] // REASON: inconsistent ENTRYPOINT_NOT_FOUND panic message
```
Comment (andrew-fleming) [inline src/tests/upgrades/test_upgradeable.cairo:80]:
```suggestion
#[ignore] // REASON: should_panic attribute not fit for complex panic messages.
```
For consistency with #1049

**Review by andrew-fleming — COMMENTED** [review]
Looking good! I left some minor suggestions and a question

**Review by ericnordelo — APPROVED** [review]
I will fix the pending comments in another PR, since this is blocking presets test migration.

**Pull request merged**